src/nistEncode.ts

```diff
@@ -79,6 +80,15 @@ const invokeFormatters = ({
   return success(undefined);
 };
 
+const defaultInformationWriter = (
+  informationItem: Exclude<NistInformationItem, undefined>,
+): Buffer => {
+  if (typeof informationItem === 'string') {
```

Comment:
A doubt about this condition: in all contexts where `defaultInformationWriter` is used, the `informationItem` is always a string.
Please could you comment under which circumstances the `informationItem` could be something else than a `string`?
Is the condition perhaps not necessary, leading to a simplification?

Reply:
You're correct, the condition is unnecessary as it is already established that the `informationItem` is  a `string` at that point. I've removed it